src/Microsoft.AspNetCore.SignalR/IHubClients`T.cs

```diff
@@ -11,6 +11,8 @@ public interface IHubClients<T>
     {
         T All { get; }
 
+        T AllExcept(IReadOnlyList<string> excludedIds);
```

Comment:
Thoughts on making `IHubClients` inherit from `IHubClients<T>` where `T` is `IClientProxy`?

@davidfowl @moozzyk

Reply:
(Sorry for butting in.)
Looks like a sensible change to me, though that doesn't count for much. If this happens, might it make sense to change IHubContext at the same time to add an additional generic parameter? I had added this as a new interface on PR #783, but I suspect it would be tidier to instead replace the existing one, or just to have the existing one extend this new one:

```
public interface IHubContext<THub, T>
	where THub : Hub<T>
	where T : class
{
	IHubClients<T> Clients { get; }

	IGroupManager Groups { get; }
}
```

Reply:
I think it makes sense to derive `IHubClients` from `IHubClients<T>`

Reply:
Yeah, we can do that. That essentially leaves the `IHubClients` file empty though

Reply:
It's goodness - you don't need to remember to update it...

Reply:
Good point 😄

Reply:
Take inspiration from what old SignalR did. I believe it did the exact same thing for the exact same reasons